udata/utils.py

```diff
@@ -199,9 +200,10 @@ def recursive_get(obj, key):
     return recursive_get(value, parts) if parts else value
 
 
-def unique_string(length=None):
+def unique_string(length=36):
     '''Generate unique string'''
-    string = str(uuid4())
+    # We need a string longer than length and an UUID is 36 chars length
+    string = str(uuid4()) * int(math.ceil(length / 36.0))
     return string[:length] if length else string
```

Comment:
Which is not imported and as it's not a builtin, there is no collision.
I changed for `uuid` but it's also a standard module and the result is not an uuid but just a string,
so I propose to keep string as it's explicit